data/uk/scotland/argyll-bute/argyll-bute.csv

```diff
@@ -0,0 +1,6 @@
+id,name,brewery_type,street,address_2,address_3,city,county_province,postal_code,website_url,phone,created_at,updated_at,country,longitude,latitude,tags
+fyne-ales,Fyne Ales,micro,"","","","","","","","","","","","","",""
```

Comment:
Not sure if this PR is still a work in progress, but I think we want to at least include city, country, updated_at, and created_at for all new breweries. Thanks!

Reply:
No worries! 

Also, no need to close the PR if you're still working on it. Rename the title "(wip) Update argyll bute" or add a Work in Progress label (just added) and I'll ignore :)